backend/adapter_processor/serializers.py

```diff
@@ -70,7 +72,20 @@ def to_representation(self, instance: AdapterInstance) -> dict[str, str]:
         adapter_metadata = json.loads(
             f.decrypt(bytes(instance.adapter_metadata_b).decode("utf-8"))
         )
+        # Get the adapter_instance
+        adapter_class = Adapterkit().get_adapter_class_by_adapter_id(
```

Comment:
Can you please check the While each time initializing Adapterkit(), Is it load all adapter classes and listed.

Reply:
Loading happens only in the startup of server.